[2023-08-30T23:23:06.536+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task permission to 509
[2023-08-30T23:23:06.537+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task'
[2023-08-30T23:23:06.579+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task permission to 509
[2023-08-30T23:23:06.580+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task'
[2023-08-30T23:23:06.720+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_trigger_dag_run_operator.start_task scheduled__2023-08-29T09:30:00+00:00 [queued]>
[2023-08-30T23:23:06.729+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_trigger_dag_run_operator.start_task scheduled__2023-08-29T09:30:00+00:00 [queued]>
[2023-08-30T23:23:06.729+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-30T23:23:06.764+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): start_task> on 2023-08-29 09:30:00+00:00
[2023-08-30T23:23:06.769+0000] {standard_task_runner.py:57} INFO - Started process 2823 to run task
[2023-08-30T23:23:06.772+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_trigger_dag_run_operator', 'start_task', 'scheduled__2023-08-29T09:30:00+00:00', '--job-id', '191', '--raw', '--subdir', 'DAGS_FOLDER/dags_trigger_dag_run_operator.py', '--cfg-path', '/tmp/tmpmhb3cr1u']
[2023-08-30T23:23:06.774+0000] {standard_task_runner.py:85} INFO - Job 191: Subtask start_task
[2023-08-30T23:23:06.839+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task permission to 509
[2023-08-30T23:23:06.839+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_trigger_dag_run_operator/run_id=scheduled__2023-08-29T09:30:00+00:00/task_id=start_task'
[2023-08-30T23:23:06.842+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_trigger_dag_run_operator.start_task scheduled__2023-08-29T09:30:00+00:00 [running]> on host 07fcf706bef2
[2023-08-30T23:23:06.954+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_trigger_dag_run_operator' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T09:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T09:30:00+00:00'
[2023-08-30T23:23:06.956+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T23:23:06.957+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "start!"']
[2023-08-30T23:23:06.965+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T23:23:06.966+0000] {subprocess.py:93} INFO - start!
[2023-08-30T23:23:06.967+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T23:23:07.007+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_trigger_dag_run_operator, task_id=start_task, execution_date=20230829T093000, start_date=20230830T232306, end_date=20230830T232307
[2023-08-30T23:23:07.064+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-30T23:23:07.110+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
